### DISCUSSION TITLE
consider adding ploting time and progress-bar to the interactive interface?

### DISCUSSION THREAD

**Discussion by jiafeng5513** [ericaltendorf/plotman#290] [Ideas]
Is anyone considering add a ploting time and progress-bar to the interactive interface? Such improvements can help users  grasp the status of the system more intuitively~

**Comment by Purple-Tentacle** [comment]
There is a progress bar in interactive mode (see the . ):
Jobs (1): [1      . 2        3       4 ]

**Reply by BasilHorowt** [reply]
This is an overview visualization of all plots I'm thinking this request is about a per-plot progress bar

**Comment by BasilHorowt** [comment]
There isn't any good way to know how long a plot will take right now. At some point the tool could accumulate stats and do a best guess of how long it will take based on previous data, but that could get complex when settings change or resources change... 

If the idea from here https://github.com/ericaltendorf/plotman/issues/297 was implemented it could be one approach
